index.js

```diff
@@ -67,29 +67,33 @@ var units = {
 	seconds: '"',
 };
 
-Coords.prototype.format = function(format, latLonSeparator) {
+Coords.prototype.format = function(format, latLonSeparator, numberDecimals) {
 	if (!format) format = 'FFf';
 	if (!latLonSeparator) latLonSeparator = ' ';
 
 	if ( Object.keys(shortFormats).indexOf(format) > -1 ) {
 		format = shortFormats[format];
 	}
 
+	if (typeof numberDecimals === 'undefined') {
+		numberDecimals = 5;
+	}
+
 	var lat = formatFor(this.latValues, (this.north) ? 'N' : 'S' );
 	var lon = formatFor(this.lonValues, (this.east) ? 'E' : 'W' );
 
 	function formatFor(values, X) {
 		var formatted = format;
 		formatted = formatted.replace(/DD/g, values.degreesInt+units.degrees);
-		formatted = formatted.replace(/dd/g, values.degrees.toFixed(5)+units.degrees);
+		formatted = formatted.replace(/dd/g, values.degrees.toFixed(numberDecimals)+units.degrees);
 		formatted = formatted.replace(/D/g, values.degreesInt);
-		formatted = formatted.replace(/d/g, values.degrees.toFixed(5));
+		formatted = formatted.replace(/d/g, values.degrees.toFixed(numberDecimals));
 		formatted = formatted.replace(/MM/g, values.minutesInt+units.minutes);
-		formatted = formatted.replace(/mm/g, values.minutes.toFixed(5)+units.minutes);
+		formatted = formatted.replace(/mm/g, values.minutes.toFixed(numberDecimals)+units.minutes);
 		formatted = formatted.replace(/M/g, values.minutesInt);
-		formatted = formatted.replace(/m/g, values.minutes.toFixed(5));
-		formatted = formatted.replace(/ss/g, values.seconds.toFixed(5)+units.seconds);
-		formatted = formatted.replace(/s/g, values.seconds.toFixed(5));
+		formatted = formatted.replace(/m/g, values.minutes.toFixed(numberDecimals));
+		formatted = formatted.replace(/ss/g, values.seconds.toFixed(numberDecimals)+units.seconds);
+		formatted = formatted.replace(/s/g, values.seconds.toFixed(numberDecimals));
 		
```

Comment:
This section could be more DRY, but original code is to blame. Can be fixed in a later code style PR.